src/scripts/main.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// write code here
+const getE = (element) => document.querySelector(element);
```

Comment:
don't cut names, use full names in the future (actually as Alisa suggested https://github.com/mate-academy/js_growth_table_DOM/pull/332#discussion_r1014605254)